package.json

```diff
@@ -83,11 +89,23 @@
         "extends": [
           "yscope/typescript"
         ]
+      },
+      {
+        "files": [
+          "test/**/*"
+        ],
+        "env": {
+          "jest": true
+        },
```

Comment:
What does this do?

Reply:
To avoid warnings on uses of the jest globals (e.g., `describe`, `test`, `expect`) without imports.